charts/routes/values.yaml

```diff
@@ -216,3 +216,14 @@ cert-manager:
 
 common:
   enabled: true
+
+############################################################
+# Oathkeeper
+############################################################
+oathkeeper:
+  # Set to false if there is an existing oathkeeper deployment
+  enabled: false
```

Comment:
since we're increasing the chart version by a minor version. I think it's safe to enable this by default. Also add other possible values here. I see a lot more in the test values

Reply:
This is a little tricky - it's actually similar to Istio, in the sense that, we don't necessarily want to Istio together with the routing chart, but we still need to enable this to test out our rules. So i have some hesitation as to whether this should be enabled by default or not.

Reply:
Should we still add the reference to the values file from the chart here as part of the documentation?

Agree on not enabling it by default. But if we have any common values for our use case we can add them here keeping the flag disabled.